recipes/awkward-pandas/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "awkward-pandas" %}
+{% set version = "2023.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/a/awkward_pandas/awkward_pandas-{{ version }}.tar.gz
+  sha256: 56b7b735241502423ac9afb49db0dd3bb5905a51a53cdfe476e9d4316a975fde
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - hatchling >=1.8.0
+    - hatch-vcs
+    - toml
+  run:
+    - python >=3.8
+    - awkward >=2.0.0
+    - pandas >=1.2
+    - pyarrow >=7.0.0
+
+test:
```

Comment:
You might consider adding a `pip check`:

https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L58-L64

Reply:
thanks!